o.js/o.js.d.ts

```diff
@@ -0,0 +1,83 @@
+// Type definitions for o.js v0.2.2
+// Project: https://github.com/janhommes/o.js
+// Definitions by: Matteo Antony Mistretta <https://github.com/IceOnFire>, Brad Zacher <https://github.com/bradzacher>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="../q/Q.d.ts" />
+
+interface Options {
```

Comment:
http://definitelytyped.org/guides/best-practices.html#ghost-modules
please check this document.